packages/progress-bar/src/ProgressBar.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Heading } from "@kaizen/component-library"
+import React, { ReactNode } from "react"
+import classnames from "classnames"
+import styles from "./ProgressBar.scss"
+
+type Props = {
+  value: number
+  max: number
+  animating: boolean
```

Comment:
I wonder if we should make this prop name consistent with Illustration's `isAnimated`

Reply:
it seems like a different thing to me. "animating" is the current state of whether the bar is . It would literally be a bug if the consumer didn't stop animating the the progress bar after it reaches 100% (which consumers will probably do).  `isAnimated` sounds like the progress bar will do this for them, but it won't.